brainsets/core.py

```
@@ -1,146 +1,19 @@
-    "NestedEnumType",
-    "StringIntEnum",
-    "Dictable",
-]
 
-_constants = [
-    "serialize_fn_map",
-]
+_constants = ["serialize_fn_map"]
 
-__all__ = _functions + _classes + _constants
+__all__ = _functions + _constants
```

Ruff (0.15.14)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)